### PR TITLE
Add TLSv1.3 draft-19 messages to trace

### PR DESCRIPTION
**Pull request by tmshort**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Comment by levitte** [comment]
Feel free to merge, @richsalz

**Comment by richsalz** [comment]
doing it now.

**Comment by richsalz** [comment]
07518cf  thanks.

File: apps/s_cb.c [hunk 1]
```diff
@@ -522,24 +522,29 @@ static STRINT_PAIR alert_types[] = {
...
     {", CertificateUrl", 21},
-    {", CertificateStatus", 22},
+    {", CertificateStatus", SSL3_MT_CERTIFICATE_STATUS},
     {", SupplementalData", 23},
-    {", KeyUpdate", 24 },
+    {", KeyUpdate", SSL3_MT_KEY_UPDATE},
+#ifndef OPENSSL_NO_NEXTPROTONEG
+    {", NextProto", SSL3_MT_NEXT_PROTO},
+#endif
+    {", MessageHash", SSL3_MT_MESSAGE_HASH},
```
Comment (mattcaswell) [inline apps/s_cb.c]: This is a "dummy" message type. It never appears on the wire. It is only ever used in calculating the transcript hash

File: ssl/t1_trce.c [hunk 2]
```diff
@@ -97,7 +98,11 @@ static ssl_trace_tbl ssl_handshake_tbl[] = {
     {SSL3_MT_CLIENT_KEY_EXCHANGE, "ClientKeyExchange"},
     {SSL3_MT_FINISHED, "Finished"},
     {SSL3_MT_CERTIFICATE_STATUS, "CertificateStatus"},
-    {SSL3_MT_KEY_UPDATE, "KeyUpdate"}
+    {SSL3_MT_KEY_UPDATE, "KeyUpdate"},
+# ifndef OPENSSL_NO_NEXTPROTONEG
+    {SSL3_MT_NEXT_PROTO, "NextProto"},
+# endif
+    {SSL3_MT_MESSAGE_HASH, "MessageHash"}
```
Comment (mattcaswell) [inline ssl/t1_trce.c]: As above

Comment (tmshort) [inline ssl/t1_trce.c]: I figured, given the outlier value; but it's in draft-19

Comment (tmshort) [inline ssl/t1_trce.c]: But, is there harm in leaving it in?

Comment (mattcaswell) [inline ssl/t1_trce.c]: I guess not - in case some weird implementation does the wrong thing and actually sends it!

**Pull request closed without merging**